### ISSUE TITLE
Failed to find required executable rsync

### ISSUE THREAD

**Issue by RedlineTriad** [gofrolist/molecule-action#138]
This actually suddenly stopped working for me a few days ago with the error
`Molecule default > create`
```
   TASK [Synchronization the context] *********************************************
  failed: [localhost] (item={'image': 'debian:10', 'name': 'instance'}) => {"ansible_index_var": "i", "ansible_loop_var": "item", "changed": false, "i": 0, "item": {"image": "debian:10", "name": "instance"}, "msg": "Failed to find required executable \"rsync\" in paths: /usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"}
```

Here is a working job:
https://github.com/RedlineTriad/private_server/actions/runs/3131176997/jobs/5082243386

And here is a failing one with basically only markdown changes:
https://github.com/RedlineTriad/private_server/actions/runs/3215501015/jobs/5256673943#step:4:85

I tried to go to an older tag instead of just `v2` but it didn't fix the issue.
And I also didn't find any issues in the github action runner repo referencing something like a change to remove rsync.
Haven't looked into this too much yet (like if rsync exists) since I need to go to sleep now.

**Comment by gardar** [comment]
I'm facing this issue as well.
It seems to be caused by version 2.1 of molecule-docker, more specifically this commit: https://github.com/ansible-community/molecule-docker/pull/169 which uses the synchronize ansible module (which depends on rsync).

**Comment by gofrolist** [comment]
rsync has been added in [v2.3.0](https://github.com/gofrolist/molecule-action/releases/tag/v2.3.0) (#139)

**Comment by RedlineTriad** [comment]
Fixed: RedlineTriad/private_server#15

**Comment by RedlineTriad** [comment]
Thank you for the quick response :+1: @gofrolist
